users/views.py

```diff
@@ -18,8 +18,8 @@
 def api_root(request, format=None):
     if not request.user.is_authenticated:
         return Response({
-                'users': reverse(
-                    'users-list',
+                'registration': reverse(
```

Comment:
I don't see option for registration at `localhost:8000` when I am not logged in.